src/Microsoft.AspNet.WebHooks.Common/Microsoft.AspNet.WebHooks.Common.nuspec

```diff
@@ -6,9 +6,9 @@
     <title>Microsoft ASP.NET WebHooks Common Module</title>
     <authors>Microsoft</authors>
     <owners>Microsoft, aspnet</owners>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
-    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=690277</projectUrl>
-    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <licenseUrl>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <projectUrl>https://go.microsoft.com/fwlink/?LinkId=690277</projectUrl>
```

Comment:
This URL redirects to the [ASP.NET WebHooks overview](https://docs.microsoft.com/en-us/aspnet/webhooks/) page in our documentation. We have a build task that replaces this with http://www.asp.net/ (yes, not HTTPS ☹️ -- separate bug) and our ASP.NET Core WebHooks projects use https://asp.net. Which of these three do we prefer in the ASP.NET WebHooks .nuspec files?

Reply:
The FWLink sounds fine, so long as it actually stays there. Can we fix the thing that overwrites it?

Reply:
👍 I've updated a related issue on "the thing" to include this fix.